### ISSUE TITLE
Official kRO Malangdo Card Remover

### ISSUE THREAD

**Issue by Forte22** [rathena/rathena#5265]
* **rAthena Hash**: da95fc7

* **Client Date**:  2020

* **Server Mode**: Renewal

* **Description of Issue**: Richard in Malangdo needs a rework.
  * Result:  Asks for 1m Zeny and has a chance to destroy equip or cards.
  * Expected Result: Ask for 100k zeny per attempt and you can choose the slot you want to remove the card from. 5% chance. No chance to destroy the card or equipment.
  * How to Reproduce: Try to remove a card in malangdo 
![grafik](https://user-images.githubusercontent.com/66630784/88882962-58035580-d233-11ea-9de7-343dc84e466a.png)

  * Official Information: 

https://youtu.be/YeYMgAlSTc4

**Comment by aleos89** [comment]
@eppc0330 The PR you created, are all of these changes reflected on kRO?

**Comment by eppc0330** [comment]
> @eppc0330 The PR you created, are all of these changes reflected on kRO?

Yes it is following official, but that PR is incompleted and need more clean up that at that time I didnt know well about scripting.
Currently I have finished clean up for my server but still not finished updating list of boss cards.
